app/models/link_popularity.rb

```diff
@@ -1,11 +1,16 @@
+# frozen_string_literal: true
+
 class LinkPopularity
   extend LogstashPrefix
+
   def self.popularity_for(url, days_back)
     link_popularity_query = ElasticLinkPopularityQuery.new(url, days_back)
-    total = ES::ELK.client_reader.count(index: indexes_to_date(days_back, true), type: 'click', body: link_popularity_query.body)["count"]
```

Comment:
The only change here (other than wrapping the long line) is the removal of `type: 'click'` from the args passed to the ES client search, which is deprecated in ES 6.